Ambit/Source/Ambit/Mode/TestClasses/MockableConfigImportExport.h

```diff
@@ -21,81 +21,62 @@
 
 #include "MockableConfigImportExport.generated.h"
 
+/**
+ * Mock class for ConfigImportExport.h
+ *
+ * Currently the Lambda implementation is being used for unit tests. This needs to be updated in the future under a new
+ * story.
+ */
 UCLASS()
-class UMockableConfigImportExport : public UConfigImportExport
+class UConfigImportExportMock : public UObject, public IConfigImportExport
 {
     GENERATED_BODY()
 public:
-    /**
-     * Overrides the default behavior of LambdaGetPathFromPopup, the function called that creates a popup for writing a file to disk,
-     * to be the function passed in.
-     */
-    void SetMockGetPathFromPopup(TFunction<FString(const FString& FileExtension, const FString& DefaultPath,
-                                                   const FString& FileName)> MockFunction)
+    FReply OnImportSdf() override
     {
-        LambdaGetPathFromPopup = std::move(MockFunction);
-    };
+        return bHandled ? FReply::Handled() : FReply::Unhandled();
```

Comment:
similar comment here: we can move the implementation to .cpp file.

Reply:
There is a reason Mock classes don't have a corresponding cpp file:
1. Each function is a single line return. Creating a cpp file for these could potentially lead to file bloat as Ambit grows.
2. Once gMock is introduced, it will be reduced back to a single header file anyway

Reply:
I agree with Rushil's decision here.